### PR TITLE
"Updated R4Practitioner to support last updated."

### PR DESCRIPTION
**Pull request by bcastillomartel-va**

### CODE REVIEW

**Comment by jblefkove-va** [comment]
The merge conflict will be OBE after https://github.com/department-of-veterans-affairs/health-apis-data-query/pull/960 merges

File: data-query-tests/src/test/java/gov/va/api/health/dataquery/tests/r4/PractitionerIT.java [hunk 1]
```diff
@@ -208,6 +249,13 @@ void searchByName() {
             b -> isNotEmpty(b),
             "Practitioner?name={family}",
             testIds.practitioners().family()),
+        test(
+            200,
+            Practitioner.Bundle.class,
+            b -> isNotEmpty(b),
+            "Practitioner?name={family}&_lastUpdated={lastUpdated}",
```
Comment (jblefkove-va) [inline data-query-tests/src/test/java/gov/va/api/health/dataquery/tests/r4/PractitionerIT.java:256]: Isn't this test case already covered in the other method?

Comment (bcastillomartel-va) [reply data-query-tests/src/test/java/gov/va/api/health/dataquery/tests/r4/PractitionerIT.java:256]: I'll remove it when and fix the merge conflicts

**Review by jblefkove-va — APPROVED** [review]
:shipit:

**Pull request merged**